webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiConversations.js

```diff
@@ -451,7 +447,7 @@ export class SakaiConversations extends SakaiElement {
         ${renderBackButton ? html`
         <div id="conv-back-button-block">
           <div>
-            <a href="javascript:;" @click="${this._setStateNothingSelected}">
+            <a href="javascript:;" @click=${this._setStateNothingSelected} aria-label="${this._i18n.back}">
```

Comment:
can we use button instead? I can't find any source that says doing `href="javascript:;" is a good thing

Reply:
I totally agree, but I didn't really want to do a load of refactoring in this PR - I wanted to focus on tests. I think a separate JIRA focusing on semantic markup in the conversations tool would be the place to do this. But yes, I totally agree.